cumulusci/robotframework/Salesforce.py

```diff
@@ -238,26 +241,29 @@ def click_object_button(self, title):
         self._jsclick(locator)
         self.wait_until_modal_is_open()
 
-    def load_related_list(self, heading):
+    @capture_screenshot_on_error
+    def load_related_list(self, heading, tries=10):
         """Scrolls down until the specified related list loads.
+
+        If the related list isn't found, the keyword will scroll down
+        in 100 pixel increments to trigger lightning into loading the
+        list. This process of scrolling will be repeated until the
+        related list has been loaded or we've tried several times
+        (the default is 10 tries)
```

Comment:
Are you sure 10 is going to be enough? It was previously 50. There can be pages with quite a few related lists.

Reply:
No, I'm not 100% sure. Though I think for our testing scenarios it probably is. However, now that it's parameterized, if a tester knows they are testing a page with a lot of data then they can set it to a very large number. I can easily bump the default number back up if you think I should. 50 just seemed like overkill, at least for the tests I was running. 

Ideally the code would scroll until it discovers there's nothing left to scroll rather than scrolling a fixed number of times. That was going to require more work and more testing, and I didn't want to hold up this fix for that.